interni/verbali/VI_2021_02_01/res/contenuto.tex

```diff
@@ -24,7 +23,9 @@ \subsection*{Presenze}
 \end{itemize}
 
 \section*{Svolgimento}
-\subsection*{Punto ordine del giorno}
-\subsection*{Punto ordine del giorno}
-\subsection*{Punto ordine del giorno}
-\subsection*{Punto ordine del giorno}
+\subsection*{Discussione valutazione RR}
+Viene discussa la correzione della RR, identificando i punti critici e possibili domande da porre per chiarimenti.
+\subsection*{Possibili correzioni}
+Vengono discusse delle possibili correzioni dei documenti. Viene identificata la necessità di un colloquio per delucidazioni sui punti critici.
+\subsection*{Varie ed eventuali}
+Viene discusso in che modo affrontare la prossima revisione, in particolare viene discusso in modo generale in che modo impostare la \glock{tecnology baseline} ed il \glock{proof of concept} del prodotto.
```

Comment:
Al posto di: Viene discusso in che modo affrontare la prossima revisione, in particolare viene discusso in modo generale in che modo impostare la \glock{tecnology baseline} ed il \glock{proof of concept} del prodotto.

Viene discusso in che modo affrontare la prossima revisione, in particolare come impostare la \glock{tecnology baseline} ed il \glock{proof of concept} del prodotto.